### ISSUE TITLE
[BUG] Enabling the GPU via GUI is not working

### ISSUE THREAD

**Issue by pete122** [harvester/harvester#6217]
**Describe the bug**
After clicking "Enable" in the SR-IOV GPU part of UI, the GPU's still remains in "Disabled" state.

**To Reproduce**
1. Download required GPU driver from NVIDIA website
2. Extract desired file and set up local web server
![webserver](https://github.com/user-attachments/assets/c31fe6c0-0056-4744-a414-c46c2129a8f5)
(* both NVIDIA files are identical, it is just a copy with shorter filename)
3. Go to Addons page and enable "nvidia-driver-toolkit"
![nvidia-driver-toolkit](https://github.com/user-attachments/assets/3c7c11cd-0d37-4b28-8f4f-05dda36bb41c)
4. Go to SR-IOV GPU Devices page of UI, select three vertical dots in the right side of the screen and click "Enable"
![sr-iov_gpu_devices](https://github.com/user-attachments/assets/42fe78ac-cb9d-4924-acd7-2ec61948b5ee)

**Expected behavior**
GPU should be in the "Enabled" state.

**Support bundle**
Will be sent to harvester-support-bundle@suse.com with assigned Issue ID.

**Environment**
 - Harvester ISO version: 1.3.1
 - Underlying Infrastructure: 2x Baremetal DELL C6620 with NVIDIA A2 GPU

**Additional context**
None

**Comment by ibrokethecloud** [comment]
are you please able to confirm that the `nvidia-driver-runtime` daemonset is running in harvester-system namespace? The image `rancher/harvester-nvidia-driver-toolkit` is not packaged in the harvester iso and needs to be downloaded in case you are in  an airgapped environment.

**Comment by pete122** [comment]
Yes I can confirm, that nvidia-driver-runtime is running and BTW, it is NOT airgapped environment.
![image](https://github.com/user-attachments/assets/e5d5bcc3-0b5a-48a6-ad77-3dc839a3569b)

**Comment by ibrokethecloud** [comment]
the driver package does not seem to have `sriov-manage` command.

```
2024-07-22T15:37:34.544656462Z time="2024-07-22T15:37:34Z" level=error msg="/bin/sh: /usr/lib/nvidia/sriov-manage: No such file or directory\n"
2024-07-22T15:37:34.544708726Z time="2024-07-22T15:37:34Z" level=error msg="error syncing 'agrotrend1-000017000': handler on-gpu-change: error performing sriovmanage operation: error during command execution: command terminated with exit code 127, requeuing"
```
are you able to please exec into one of the pods below in `harvester-system` namespace and provide us the output of `ls -lR /usr/lib/nvidia`?

```
nvidia-driver-runtime-k7n7r                             1/1     Running     0             3d9h
nvidia-driver-runtime-qqcgl                             1/1     Running     0             3d9h
```

**Comment by pete122** [comment]
![image](https://github.com/user-attachments/assets/45fe4fbb-43e2-465c-9861-d693c750edc3)
The "sriov-manage" file seems to be there. Should I try to restart both nodes to see, if it will change anything?

**Comment by pete122** [comment]
BTW commands works only on the first (Management) Node"agrotrend**1**". Trying to get pods on the second (Worker) Node "agrotrend**3**", will result in failure. Is such a behavior OK?
![image](https://github.com/user-attachments/assets/0f771098-9fe5-447a-bc80-3782d02f2ca2)

**Comment by ibrokethecloud** [comment]
the command can only be run from the controlplane nodes, I believe apart from the first node you assigned all other nodes a worker role. you can still exec into pods on any node from the controlplane node. you do not ned to ssh to each node to check the contents of the package

**Comment by pete122** [comment]
> the command can only be run from the controlplane nodes, I believe apart from the first node you assigned all other nodes a worker role. you can still exec into pods on any node from the controlplane node. you do not ned to ssh to each node to check the contents of the package

OK, thanks for explanation 👍

**Comment by pete122** [comment]
> ![image](https://private-user-images.githubusercontent.com/176375838/352556322-45fe4fbb-43e2-465c-9861-d693c750edc3.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjI4NjcwMDYsIm5iZiI6MTcyMjg2NjcwNiwicGF0aCI6Ii8xNzYzNzU4MzgvMzUyNTU2MzIyLTQ1ZmU0ZmJiLTQzZTItNDY1Yy05ODYxLWQ2OTNjNzUwZWRjMy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwODA1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDgwNVQxNDA1MDZaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1kMWE4YWUyMzMyZDkwYTBjNGMyYjZjNzUwZjE3MTJjYTYzZmRkZWQxYzYwNDU2Mjc0ZTNhNTE1MzQ4ZmQ5ZTUzJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.YiUZPsKeIkVgiuItUuWxNij0daP1ul1B6-BLMsPCe8c) The "sriov-manage" file seems to be there. Should I try to restart both nodes to see, if it will change anything?

Hi folks, any new idea, how to proceed?

**Comment by pete122** [comment]
New Harvester version 1.3.2 has been released. Should I try to upgrade? Will it solve the problem?
